### DISCUSSION TITLE
Multiplayer leaderboard doesn't update during map

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21548] [Q&A] [answered]
I was playing osu!lazer multi and my game wouldn't actually update any of our scores until the map ended. Other people in the multi also reported having the same issue.

![image](https://user-images.githubusercontent.com/74088315/206089311-7cd79e02-34e1-4ed8-ab99-a006ace9fb0d.png)

[performance.log](https://github.com/ppy/osu/files/10172801/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10172802/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10172803/updater.log)
[database.log](https://github.com/ppy/osu/files/10172804/database.log)
[network.log](https://github.com/ppy/osu/files/10172805/network.log)

**Comment by smoogipoo** [comment, marked as answer]
A fix has been deployed, but will only affect newly-created rooms.
